src/app/components/CartButton.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+import React from 'react';
+import { useCart } from '@/app/contexts/CartContext';
+
+const CartButton = ({ product }) => {
+  const { addToCart } = useCart(); //context
+
+  const handleAddToCart = () => {
+    addToCart(product);
+    window.alert("Product added succesfully")
```

Comment:
You could implement something like https://www.npmjs.com/package/react-toastify for notifications

Reply:
Even the `addToCart` function from context could be the one that triggers the notification.